src/CSModule.sol

```diff
@@ -990,9 +991,10 @@ contract CSModule is
     /// @dev SETTLE_EL_REWARDS_STEALING_PENALTY_ROLE role is expected to be assigned to Easy Track
     /// @param nodeOperatorIds IDs of the Node Operators
     function settleELRewardsStealingPenalty(
-        uint256[] memory nodeOperatorIds
+        uint256[] calldata nodeOperatorIds
     ) external onlyRole(SETTLE_EL_REWARDS_STEALING_PENALTY_ROLE) {
-        for (uint256 i; i < nodeOperatorIds.length; ++i) {
+        uint256 noIdCount = nodeOperatorIds.length;
```

Comment:
Double check gas usage.

Reply:
Without this change, it is even cheaper. lol